cmd/server/server.go

```diff
@@ -33,6 +37,10 @@ func parseArguments() cliArguments {
 		"JSON file containing proxies to create on startup")
 	flag.Int64Var(&result.seed, "seed", time.Now().UTC().UnixNano(),
 		"Seed for randomizing toxics with")
+	flag.BoolVar(&result.runtimeMetrics, "enable-runtime-metrics", false,
+		`enable runtime-related prometheus metrics (default "false")`)
+	flag.BoolVar(&result.proxyMetrics, "enable-proxy-metrics", false,
+		`enable toxiproxy-specific prometheus metrics (default "false")`)
```

Comment:
Should these flag names change to just `-runtime-metrics` and `-proxy-metrics`? The `enable` piece probably isn't necessary?